impl/src/main/java/org/ehcache/internal/store/offheap/OffHeapValueHolder.java

```diff
@@ -33,14 +33,15 @@
   private final V value;
   private final WriteContext writeContext;
 
-  public OffHeapValueHolder(long id, V value, long creationTime, long expireTime) {
-    this(id, value, creationTime, expireTime, 0, null);
+  public OffHeapValueHolder(long id, V value, long creationTime, long expireTime, long hits) {
```

Comment:
If this is always called with zero (since new values have no hits) then we can eliminate the param and hardcode the zero, right?